sessionctx/variable/tidb_vars.go

```diff
@@ -344,6 +347,7 @@ const (
 	DefTiDBForcePriority               = mysql.NoPriority
 	DefTiDBUseRadixJoin                = false
 	DefEnableWindowFunction            = true
+	DefEnableVectorizedExpression      = false
```

Comment:
```suggestion
	DefEnableVectorizedExpression      = true
```

So it can be tested in the master branch.

Reply:
fixed.